netx/net/sourceforge/jnlp/runtime/Boot.java

```diff
@@ -343,6 +343,14 @@ private static String getMainFile() throws InvalidArgumentException {
         return null;
     }
 
+    private static String stripDoubleQuote(String path) {
```

Comment:
how about put private helper methods to the end of a class (behind public methods etc.), Just my personal taste ;-)

Reply:
+1